src/main/java/todolist/service/UserManager.java

```diff
@@ -0,0 +1,40 @@
+package todolist.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import todolist.model.User;
+import todolist.repository.UserRepository;
+
+@Service
+public class UserManager {
+
+    public static User user;
+
+    @Autowired
+    public UserRepository userRepository;
+
+    public void userRegistration(User user){
+        userRepository.save(user);
+    }
+
+    public String loginValidation(User user){
```

Comment:
to tu nie należy, utwórz osobną klasę walidatora dla użytkownika i niech nie zwraca ona templatek, tylko np rzuca wyjątki które będziesz mógł obsłużyć, a w przypadku ich braku niech kontroler templatkę wyrenderuje